supervisor/daemon/wireguard/wginterface/firewall/helpers.go

```diff
@@ -147,3 +152,24 @@ func getCurrentProcessAppID() (*wtFwpByteBlob, error) {
 	}
 	return appID, nil
 }
+
+func getMystAppID() (*wtFwpByteBlob, error) {
+	currentFile, err := os.Executable()
+	if err != nil {
+		return nil, wrapErr(err)
+	}
+
+	currentFile = filepath.Join(filepath.Dir(currentFile), MYST_EXECUTABLE)
```

Comment:
Is this really the desired outcome?

![image](https://user-images.githubusercontent.com/20511486/127615424-86ff4d59-694c-485d-91f5-92d63312db33.png)

Reply:
@mdomasevicius You've missed `filepath.Dir`

Reply:
@mdomasevicius https://play.golang.org/p/glK_ID7yTDJ